packages/expo-modules-core/e2e/setup/create-eas-project-macos.ts

```diff
@@ -0,0 +1,363 @@
+import spawnAsync from '@expo/spawn-async';
+import { rmSync, existsSync } from 'fs';
+import fs from 'fs/promises';
+import nullthrows from 'nullthrows';
+import path from 'path';
+
+const dirName = __dirname;
+
+// Package dependencies in chunks based on peer dependencies.
+const dependencyChunks = [
+  ['@expo/config-types', '@expo/env'],
+  ['@expo/config'],
+  ['@expo/config-plugins'],
+  ['expo-modules-core'],
+  ['@expo/cli', 'expo', 'expo-asset', 'expo-modules-autolinking'],
+  ['@expo/prebuild-config', '@expo/metro-config', 'expo-constants', 'expo-manifests'],
+  [
+    'babel-preset-expo',
+    'expo-application',
+    'expo-device',
+    'expo-eas-client',
+    'expo-file-system',
+    'expo-font',
+    'expo-json-utils',
+    'expo-keep-awake',
+    'expo-splash-screen',
+    'expo-status-bar',
+    'expo-structured-headers',
```

Comment:
Honestly `expo-modules-core` package doesn't seem to be the right place for these setup scripts. The project is actually using a bunch of modules, not just the modules core 🤔

Reply:
We could clean this up to just test some key packages, just not sure to which folder we should move this to

Reply:
not sure if that's a good idea. if the fixtures are not changed frequently, maybe we can put it in the other git repo? we can maintain these custom actions like https://github.com/expo/eas-custom-builds-example/tree/main/.eas/build

also we may able to automatically retrieve the tested package from parsing expo-module.config.json and search for `apple`.

Reply:
alternatively, since we only test building and without e2e testing. maybe the other idea is like [create-expo-nightly](https://github.com/expo/expo/tree/main/packages/create-expo-nightly) and its [ci workflow](https://github.com/expo/expo/blob/main/.github/workflows/test-react-native-nightly.yml). we can also enhance create-expo-nightly for macos testing if necessary.